LocalizeNIB/LocalizeNIB.swift

```diff
@@ -27,49 +29,60 @@ public protocol Localizable {
 
 /// Implements localization logic. Use shared instance for customizing behaviour of this class.
 open class LocalizeNIB {
-    open static var instance = LocalizeNIB()
-    
-    /// String provider used for localization. Set to nil to provide default implementation that simply calls NSLocalizedString.
+    public static var instance = LocalizeNIB()
+
+    /// String provider used for localization.
+    /// Set to nil to provide default implementation that simply calls NSLocalizedString.
     open var stringProvider: LocalizedStringProvider?
-    
-    /// If set this block is called before localizing each object. If it returns true the localization will not be performed on this object. Default value is nil.
+
+    /// If set this block is called before localizing each object.
+    /// If it returns true the localization will not be performed on this object. Default value is nil.
     open var localizeAll: LocalizerBlock?
-    
+
     /// If true, each object that is not implementing Localizable protocol will be reported. Default is false
     open var debugMode: Bool
-    
+
     /// This block is called for each debug message. The default implementation simply calls print.
     open var debugBlock: ((String) -> Void) = { print($0) }
-    
+
+    /// A special string to report that some value for key is missing
+    open var missingKeyIdentifier: String { return "!!!" }
+
     fileprivate lazy var defaultStringProvider: LocalizedStringProvider = {
         return { [weak self] string in
-            let result = NSLocalizedString(string, value: "!!!", comment: "") // will return !!! if string is not found in string table
-            if result == "!!!" {
-                if self?.debugMode == true {
-                    self?.debugBlock("Missing key '\(string)'")
+            guard let strongSelf = self else {
+                return String()
```

Comment:
This actually changes behavior of the provider. It was written in a way that it can work without strong reference to parent (required only for debugMode check)

Reply:
Yes, it was my mistake, but I fixed it now.